docs/book/limits.md

```diff
@@ -0,0 +1,10 @@
+# vSphere CSI Driver - Limits
+
+| Limits                                              | vanilla block                     | vanilla File                         | WCP supervisor | WCP Guest    |
```

Comment:
WCP supervisor -> Pacific supervisor cluster
WCP Guest -> Tanzu Kubernetes Grid

Reply:
Let me address this in the another PR.